src/app/reducers/references.handlers.ts

```diff
@@ -2,71 +2,87 @@ import { Node as ProsemirrorNode } from 'prosemirror-model';
 
 import { ManuscriptHistoryState } from 'app/store';
 import { getRefNodeText, Reference, sortReferencesList } from 'app/models/reference';
-import { cloneManuscript } from 'app/utils/state.utils';
-import { createDiff, updateManuscriptState } from 'app/utils/history.utils';
+import { UpdateObjectChange } from 'app/utils/history/update-object-change';
+import { ProsemirrorChange } from 'app/utils/history/prosemirror-change';
+import { BatchChange } from 'app/utils/history/change';
+import { AddObjectChange } from 'app/utils/history/add-object-change';
+import { RearrangingChange } from 'app/utils/history/rearranging-change';
+import { DeleteObjectChange } from 'app/utils/history/delete-object-change';
 
 export function updateReference(state: ManuscriptHistoryState, payload: Reference): ManuscriptHistoryState {
   const referenceIndex = state.data.present.references.findIndex(({ id }) => id === payload.id);
+  const referenceChange = new UpdateObjectChange(
```

Comment:
If I change the order of the authors in a reference, when i return to the main article view that reference is now out of place in the list. My guess is that it isn't re-triggering a sort when the reference changes.